.github/workflows/ci-javascript.yaml

```diff
@@ -272,3 +272,65 @@ jobs:
       - name: Test CLI JS
         run: yarn test
         working-directory: ./packages/js/cli
+
+  Test-Cli-Standalone-Package:
```

Comment:
i think we should create three different jobs that run in each platform we want to support, `macos-latest`, `windows-latest` and `ubuntu-latest`; and run `yarn test:pkg`

another interesting approach: build a wrapper without having node js installed? just thinking out loud here :yum:

Reply:
I will work on this on Monday

Reply:
NOTE: running "macos" & "windows" containers in CI was not possible last time I checked. We need to spin-up docker containers within our CI, and this isn't supported by these container options on GitHub (or at least it wasn't in the past).

But I agree we are definitely lacking e2e tests for mac & windows, should see if GitHub has improved this, could be better supported now :P

Reply:
they actually support it now, in swift client we use `macos-latest` and `windows-latest` is also supported https://github.blog/changelog/2022-01-11-github-actions-jobs-running-on-windows-latest-are-now-running-on-windows-server-2022/

Reply:
Ah nice, but do they support running linux docker containers within those containers? That was the problem we ran into last time.